gekitchen/erd_utils.py

```diff
@@ -77,6 +76,18 @@ def _decode_erd_string(value: str) -> str:
 
     return raw_bytes[1:].decode('ascii')
 
+def _decode_erd_string(value: str) -> str:
+    """
+    Decode an string value sent as a hex encoded string.
+
+    TODO: At least for the dishwasher cycle the first character is not a check sum
+    are there potentially different decoding methods needed?
```

Comment:
Looks like you're correct.  I went back to the decompiled java courtesy of [bytecode-viewer](https://github.com/Konloch/bytecode-viewer) on the v1.0.3.13 APK, and found this:
![image](https://user-images.githubusercontent.com/5182431/102702035-e2247880-422b-11eb-8d43-1534893259f1.png)

Looks like we can kill that TODO, and I'll open an issue for myself on the other.

Reply:
Sounds good - I've got another (bigger) update coming, we can resolve in that one